app/main/views/drafts.py

```diff
@@ -350,6 +353,10 @@ def delete_draft_service(draft_id):
         db_object=None
     )
 
+    audit.acknowledged = True
+    audit.acknowledged_by = "acknowledged_at_creation"
```

Comment:
I'd put something like `acknowledging service copy reset` (which is what is happening).

Reply:
Good idea